Core/DISET/private/BaseClient.py

```diff
@@ -252,9 +254,10 @@ def _connect( self ):
       if not retVal[ 'OK' ]:
         if self.__retry < 5:
           url = "%s://%s:%d/%s" % ( self.__URLTuple[0], self.__URLTuple[1], int( self.__URLTuple[2] ), self.__URLTuple[3] )
-          if url not in self.__bannedUrls: 
-            gLogger.notice( "URL banned", "%s" % url )
-            self.__bannedUrls += [url]   
+          if url not in self.__bannedUrls:
+            self.__bannedUrls += [url]
+            if len( self.__bannedUrls ) < self.numberOfURLs:
```

Comment:
I am not sure to understand the logic here. The idea is that we printout a message only once we reached the max amount of trial for a given URL ?